.gitignore

```diff
@@ -0,0 +1,22 @@
+# OSX trash
+.DS_Store
```

Comment:
I'll add a trailing slash here.

Reply:
According to [this](https://github.com/github/gitignore/blob/master/Global/macOS.gitignore), perheps .DS_Store is enough.

Reply:
Ah indeed, looks like `.DS_Store` is a file and not a directory.